torch/_subclasses/fake_tensor.py

```diff
@@ -720,7 +720,11 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
 
         # Some attribute queries that can be serviced directly
         # See Note [is_coalesced is dispatched]
-        if func in [torch.ops.aten.is_coalesced.default]:
+        if func in [
```

Comment:
nit: should we get in the habit to use sets for these things? Even for the test skips, we're pretty inconsistent and use large-ish lists sometimes.

Reply:
According to SO sets are always faster than lists, so yes, let's do it